buildrunner/docker/importer.py

```diff
@@ -41,7 +43,11 @@ def import_image(self):
         """
 
         try:
-            import_return = self.docker_client.import_image(self.src)
+            import_return = None
+            if BuildRunnerConfig.get_instance().run_config.use_legacy_builder:
+                import_return = self.docker_client.import_image(self.src)
+            else:
+                import_return = python_on_whales.docker.import_(self.src)
```

Comment:
is this accurate, `import_`?

Reply:
Sadly, yes. 
https://gabrieldemarmiesse.github.io/python-on-whales/sub-commands/image/#python_on_whales.components.image.cli_wrapper.ImageCLI.import_